liquibase/src/main/resources/changelog/0022-expand-system-table.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="b305dffc-efd6-4dd3-8b75-996b5ce0fe19" author="daniel-j-mason">
+        <addColumn tableName="system">
+            <column name="last_updated" type="TIMESTAMP WITHOUT TIME ZONE"/>
```

Comment:
I would prefer to use `timestamp` since that is the liquibase type that will be translated to other systems.

additionally why would timezone not be needed / wanted here?